browser-sign-in/build.gradle

```diff
@@ -1,19 +1,73 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```

Comment:
This used to be browser-sign-in/app/build.gradle. browser-sign-in/build.gradle is moved to samples-android/build.gradle